content/applications/marketing/marketing_automation.rst

```diff
@@ -1,121 +1,201 @@
+   <email_marketing>` app, as most features of Odoo *Marketing Automation* are dependent on that
+   specific application.
 
-To add a more specific filter to a marketing automation campaign, click the :guilabel:`Add
-condition` button in the :guilabel:`Filter` field. Doing so reveals a series of other configurable
-filter rule fields.
+   Additionally, install the :doc:`CRM <../sales/crm>` and :doc:`SMS Marketing <sms_marketing>`
+   applications to access *all* of the features available in *Marketing Automation*.
 
-In the rule fields, customizable equations can be configured for Odoo to use when filtering who to
-include or exclude in this specific marketing campaign.
+   The following documentation assumes that all three of these dependent applications are installed
+   on the database.
 
-.. image:: marketing_automation/filter-node-equation-fields.png
-   :align: center
-   :alt: How the filter rule equation fields look in Odoo Marketing Automation campaigns.
+.. _marketing_automation/campaigns:
+
+Campaigns
+=========
 
-.. note::
-   :guilabel:`Records` refer to contacts in the system that fit the specified criteria for a
-   campaign.
+A *campaign* refers to a workflow of activities that are automatically executed to a target
+audience, based on predefined filters, triggers, and durations of activities.
 
-Also, once :guilabel:`Add condition` is clicked, the ability to :guilabel:`Save as Favorite Filter`
-becomes available on the campaign form.
+A new campaign can be created from scratch or from a :ref:`template
+<marketing-automation/campaign-templates>`.
 
-There is also the option to match records with :guilabel:`all` or :guilabel:`any` of the rules
-configured in the :guilabel:`Filter` field.
+To create a campaign, navigate to the :menuselection:`Marketing Automation` application, from the
+main Odoo dashboard, to open the :guilabel:`Campaigns` dashboard. From here, click the
+:guilabel:`New` button to reveal a new campaign form.
 
-To choose either of those options, click :guilabel:`all` from the middle of the sentence
-":guilabel:`Match records with all of the following rules`" to reveal a drop-down menu with those
-options.
+.. _marketing-automation/campaign-templates:
 
-.. image:: marketing_automation/match-all-any-rules-drop-down.png
+Campaign templates
+------------------
+
+Odoo provides six campaign templates to help users get started. The campaign template cards **only**
+display when there are no existing campaigns in the database. Once a campaign has been created, the
+template cards on the *Campaigns* dashboard are replaced with a Kanban view of the existing
+campaigns.
+
+To get started with a template, navigate to the :menuselection:`Marketing Automation` application,
+from the main Odoo dashboard, to open the :guilabel:`Campaigns` dashboard, which displays six
+campaign template cards:
+
+.. image:: marketing_automation/campaigns-dashboard.png
```

Comment:
Would move the image below the list, as it's meant to supplement text.
I think you can use emojis on all of the gui headers too like **🏷️ Tag Hot Contacts**,  **✌️Welcome Flow**, etc.